core/aws-lsp-core/src/test/testFolder.ts

```diff
@@ -0,0 +1,60 @@
+import * as path from 'path'
+import * as fs from 'fs'
+import * as os from 'os'
+import * as crypto from 'crypto'
+
+/**
+ * Interface for working with temporary files.
+ * Simplified port of https://github.com/aws/aws-toolkit-vscode/blob/16477869525fb79f8dc82cb22e301aaea9c5e0c6/packages/core/src/test/testUtil.ts#L77
```

Comment:
👍 for now, this kind of precise cross-reference will help a lot, in each of these util files.